parsec/core/gui/users_widget.py

```diff
@@ -313,10 +313,13 @@ def add_user(self, user_info, is_current_user):
         button.revoke_clicked.connect(self.revoke_user)
         button.show()
 
-    def add_user_invitation(self, email, invite_addr):
+    def add_user_invitation(self, email, invite_addr, greet_disabled):
         button = UserInvitationButton(email, invite_addr)
         self.layout_users.addWidget(button)
         button.greet_clicked.connect(self.greet_user)
+        button.setDisabled(greet_disabled)
+        if greet_disabled:
+            button.setToolTip(_("USERS_LIMIT_REACHED"))
```

Comment:
Given the message is about the user limit, shouldn't the parameter passed to this function be also named after that ? (i.e. there is no other reasons for greet to be disabled, and if in the future there is, we will end up with an incorrect tool tip)